### PR TITLE
code cleanup jooby-akka

### PR DESCRIPTION
**Pull request by feffi**
Just some code cleanup.

### CODE REVIEW

File: modules/jooby-akka/src/main/java/org/jooby/akka/Akka.java [hunk 1]
```diff
@@ -205,6 +205,7 @@
 
 import static java.util.Objects.requireNonNull;
 
+import com.sun.istack.internal.NotNull;
```
Comment (jknack) [inline modules/jooby-akka/src/main/java/org/jooby/akka/Akka.java]: use `javax.annotaion.Nonnull` and avoid use of internal classes

Comment (feffi) [reply modules/jooby-akka/src/main/java/org/jooby/akka/Akka.java]: Hi, it seems the JSR-305 import in the POM was missing. I added it globally in the parent POM.

Any intention from your side regarding such global POM imports? A lot of imports could be moved from modules to parent POM (e.g. PowerMock, etc.). Any thoughts on that?

Comment (feffi) [reply modules/jooby-akka/src/main/java/org/jooby/akka/Akka.java]: IntelliJ did a Fallback on NotNull in a strange way...fixed it

Comment (jknack) [reply modules/jooby-akka/src/main/java/org/jooby/akka/Akka.java]: We shouldn't add global dependencies to parent pom.xml

In core, this was handled it like:

```
<dependency>
      <groupId>com.google.code.findbugs</groupId>
      <artifactId>jsr305</artifactId>
      <scope>provided</scope>
    </dependency>
```

Comment (feffi) [reply modules/jooby-akka/src/main/java/org/jooby/akka/Akka.java]: This does work, if you add Java8EE manually to your IDE (mine hasn't this libs). Maybe I have a configiguration problem here? Any hints would be appreciated?

File: pom.xml [hunk 2]
```diff
@@ -1681,6 +1681,14 @@
     </dependencies>
   </dependencyManagement>
 
+  <dependencies>
```
Comment (jknack) [inline pom.xml]: please move this to the jooby-akka/pom.xml

**Comment by jknack** [comment]
fyi: travis build is back

**Comment by feffi** [comment]
Oh nice, how come?

**Comment by jknack** [comment]
I couldn';t figure it the easymock/javassist error while mocking URL objects. Just create a real URL object and use it. https://github.com/jooby-project/jooby/commit/25991d52e67807a1799c9f6a6bbc0be735acce30

File: pom.xml (hunk 2 above)
Comment (feffi) [reply pom.xml]: done.

**Comment by feffi** [comment]
I really like the green travis stuff :-)

**Pull request merged**